include/RAJA/index/IndexValue.hpp

```diff
@@ -334,26 +334,30 @@ constexpr RAJA_HOST_DEVICE RAJA_INLINE
   return val;
 }
 
-namespace internal{
-template<typename FROM, typename Enable = void>
-struct StripIndexTypeT {
-    using type = FROM;
+namespace internal
+{
+template <typename FROM, typename Enable = void>
+struct StripIndexTypeT
+{
+  using type = FROM;
 };
 
-template<typename FROM>
-struct StripIndexTypeT<FROM, typename std::enable_if<std::is_base_of<IndexValueBase, FROM>::value>::type>
+template <typename FROM>
+struct StripIndexTypeT<
+    FROM,
+    typename std::enable_if<std::is_base_of<IndexValueBase, FROM>::value>::type>
 {
-    using type = typename FROM::value_type;
+  using type = typename FROM::value_type;
 };
-} // namespace internal
+}  // namespace internal
```

Comment:
Adding to an earlier comment, adding a blank line before the closing curly brace of a namespace would be preferred.